app/src/main/java/io/homeassistant/companion/android/settings/SettingsFragment.kt

```diff
@@ -173,7 +173,10 @@ class SettingsFragment(
             it.entryValues = percentages.map { pct -> pct.toString() }.toTypedArray()
         }
 
-        findPreference<PreferenceCategory>("widgets")?.isVisible = Build.MODEL != "Quest"
+        val isAutomotive =
+            Build.VERSION.SDK_INT >= Build.VERSION_CODES.M && requireContext().packageManager.hasSystemFeature(PackageManager.FEATURE_AUTOMOTIVE)
+
+        findPreference<PreferenceCategory>("widgets")?.isVisible = Build.MODEL != "Quest" || !isAutomotive
```

Comment:
This will make the preference visible because you're using or instead of and. Visible if [not Quest] or [not Automotive] which can't both be true.

Reply:
🙈